server/apps/publish/formatters/aap_bulletinbuilder_formatter.py

```diff
@@ -21,6 +22,18 @@ class AAPBulletinBuilderFormatter(Formatter):
     def format(self, article, subscriber):
         try:
             pub_seq_num = superdesk.get_resource_service('subscribers').generate_sequence_number(subscriber)
+            body_html = article.get('body_html', '').strip('\r\n')
```

Comment:
This point was discussed in standup on friday. There will be separate story
for this depending on the requirement. This is just for getting sanitize
text into bulletin builder. Otherwise we r getting all sort of spell
checker marker. Moreover bulletin builder does not support html.

On Saturday, 11 July 2015, Siva Kuna <notifications@github.com> wrote:

> In server/apps/publish/formatters/aap_bulletinbuilder_formatter.py
> <https://github.com/superdesk/superdesk/pull/820#discussion_r34409605>:
>
> > @@ -21,6 +22,18 @@ class AAPBulletinBuilderFormatter(Formatter):
> >      def format(self, article, subscriber):
> >          try:
> >              pub_seq_num = superdesk.get_resource_service('subscribers').generate_sequence_number(subscriber)
> > +            body_html = article.get('body_html', '').strip('\r\n')
>
> Are we sanitizing preformatted articles also?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/superdesk/superdesk/pull/820/files#r34409605>.
>


-- 

*Mayur Dhamanwala | *Senior Analyst Programmer


*Australian Associated Press*
Level 5, 3 Rider Boulevard, Rhodes NSW 2138, Australia
PO Box 3411, Rhodes NSW 2138, Australia
*T *+61 2 9322 8582  *F *+61 2 9322 8086
mdhamanwala@aap.com.au | www.aap.com.au

-- 
*This email may contain information that is confidential. If you receive an 
email in error please delete it immediately.*